### PR TITLE
Ctctraders 449 - get all errors and display on page

### PR DESCRIPTION
**Pull request by kevincoatsworth**

### CODE REVIEW

File: app/controllers/ArrivalRejectionController.scala [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  renderer: Renderer,
+  arrivalMovementConnector: ArrivalMovementConnector,
+  frontendAppConfig: FrontendAppConfig
+)(implicit ec: ExecutionContext)
+    extends FrontendBaseController
+    with I18nSupport {
+
+  def onPageLoad(arrivalId: ArrivalId, messageId: MessageId): Action[AnyContent] = identify.async {
+    implicit request =>
+      if (frontendAppConfig.featureToggleArrivalRejection) { //TODO Feature toggle to be removed later when it is ready
```
Comment (amaalali) [inline app/controllers/ArrivalRejectionController.scala:46]: We don't need the comment for the toggle

File: app/models/messages/ErrorType.scala [hunk 2]
```diff
@@ -17,10 +17,14 @@
 package models.messages
 
 import com.lucidchart.open.xtract.{__, XmlReader}
+import play.api.libs.json.{Json, OFormat}
 
 case class ErrorType(value: Int)
 
 object ErrorType {
+
+  implicit val format: OFormat[ErrorType] = Json.format[ErrorType]
```
Comment (amaalali) [inline app/models/messages/ErrorType.scala:26]: We also only need an `OWrites` here

File: app/models/messages/FunctionalError.scala [hunk 3]
```diff
@@ -28,6 +29,8 @@ final case class FunctionalError(
 
 object FunctionalError {
 
+  implicit val format: OFormat[FunctionalError] = Json.format[FunctionalError]
```
Comment (amaalali) [inline app/models/messages/FunctionalError.scala:32]: We also only need an `OWrites` here

File: app/models/messages/ErrorPointer.scala [hunk 4]
```diff
@@ -17,10 +17,14 @@
 package models.messages
 
 import com.lucidchart.open.xtract.{__, XmlReader}
+import play.api.libs.json.{Json, OFormat}
 
 case class ErrorPointer(value: String)
 
 object ErrorPointer {
+
+  implicit val format: OFormat[ErrorPointer] = Json.format[ErrorPointer]
```
[the remark this answers is not in the document]
Comment (kevincoatsworth) [reply app/models/messages/ErrorPointer.scala]: Changed to OWrites :+1:

File: app/controllers/ArrivalRejectionController.scala (hunk 1 above)
Comment (kevincoatsworth) [reply app/controllers/ArrivalRejectionController.scala]: Removed comment :+1:

File: app/models/messages/ErrorType.scala (hunk 2 above)
Comment (kevincoatsworth) [reply app/models/messages/ErrorType.scala]: Changed to OWrites :+1:

File: app/models/messages/FunctionalError.scala (hunk 3 above)
Comment (kevincoatsworth) [reply app/models/messages/FunctionalError.scala]: Changed to OWrites :+1:

**Pull request merged**